website/docs/Installation/linux-install.md

```diff
@@ -14,31 +14,39 @@ Now that you have Podman machine up and running, you can check out the [Download
 
 ## Installing Podman Desktop
 
-### 1. Using [flatpak](https://flatpak.org/setup/)
+### 1. Using [FlatHub](https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop)
 
-Before you follow along, make sure flatpak is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up flatpak.
+With FlatHub, you can install Podman Desktop with just a single command.
 
-Open the Terminal and go to Downloads directory. 
+> Make sure to follow the [setup guide](https://flatpak.org/setup/) before installing
 
 ```sh
-cd Downloads
+flatpak install --user flathub io.podman_desktop.PodmanDesktop
 ```
+After installation, to open Podman Desktop:
 
-> In order to install & start the application, you need to know the Flatpak identifier for the application. In our case, that is `io.podman_desktop.PodmanDesktop` starting 0.0.6 and `com.github.containers.desktop` for previous versions
+```sh
+flatpak run io.podman_desktop.PodmanDesktop
+```
 
-Run the following command to install the flatpak application,
+### 2. Using [flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
```

Comment:
```suggestion
### 2. Using [flatpak](https://flatpak.org/setup/) on Linux
```

It's weird that `[Downloads]` is linked to `/downloads/linux` in this title?

Reply:
it takes you to the downloads page from where you can obtain the flatpak or the zip file